doc/source/NewSiteConfigs.rst

```diff
@@ -143,6 +143,12 @@ These instructions are meant to be a reference that users can follow to set up t
    brew install qt@5
    brew install mysql
 
+.. note::
+  On an Intel based Mac, you will need to also install pkg-config using homebrew.
+  This is done to work around an issue where libraries (eg, openssl) cannot be properly found during code compilation.
+.. code-block:: console
```

Comment:
This isn't rendered correctly (not showing up: https://spack-stack--726.org.readthedocs.build/en/726/NewSiteConfigs.html#macos)